Add "pitch shift" mod

loma-baldsson opened the pull request:
#13142

Commit:
Create mod

Commit:
Create mod

Commit:
Revert accidental change to .idea/.idea.osu.Desktop/.idea/vcs.xml

smoogipoo committed:
Revert unintended change

peppy commented:
Not sure about this one. It sounds pretty bad (as expected) since it relies on tempo shifting. And in this case doing so much more than HT/DT usually would...

Would be interested in more opinions on this one. I'd rather not have to listen to tracks with this mod applied personally.

peppy committed:
Add requires configuration flag

peppy committed:
Remove unnecessary parens

Commit:
modifié :         osu.Game/Rulesets/Mods/ModDaycore.cs
	modifié :         osu.Game/Rulesets/Mods/ModNightcore.cs
	modifié :         osu.Game/Rulesets/Mods/ModTimeRamp.cs

Mou-Ikkai commented:
> Not sure about this one. It sounds pretty bad (as expected) since it relies on tempo shifting. And in this case doing so much more than HT/DT usually would...
> 
> Would be interested in more opinions on this one. I'd rather not have to listen to tracks with this mod applied personally.

I'd say that the mod could be fun for some people. Though I think the mod might want to be a little less aggressive on the shifting. I like playing Nightcore for its pitch shifting.

loma-baldsson commented:
Gonna close this one as it doesn't look like there's really any demand for it and the implementation is a bit messy. I'll leave the branch around in case I want to come back to it later or someone else is interested in reviving it.
